media/css/base/page-not-found.scss

```diff
@@ -63,7 +63,18 @@ main {
 
 .header-image {
     padding-top: 32px;
-    width: 220px;
+    padding-right: 20px;
+    &:after {
+        content: '';
+        background: grey;
```

Comment:
We have a [set of variables for colors](https://protocol.mozilla.org/fundamentals/color.html) that help us maintain a consistent brand palette. For this I'd recommend `$color-marketing-gray-30` unless you think it should be a bit darker, then maybe try `$color-marketing-gray-40` or `$color-marketing-gray-50`